lib/extract/standalone/read_kernel_data_mod.jinja

```diff
@@ -258,6 +263,61 @@ contains
 
     end subroutine ReadArray{{dim}}d{{name}}
 
+    ! -------------------------------------------------------------------------
+    !> @brief This subroutine reads the values of a {{dim}}D array of {{type}}
+    !! that is not allocatable (e.g. a fixed size array).
+    !! @param[in,out] this The instance of the extract_PsyDataType.
+    !! @param[in] name The name of the variable (string).
+    !! @param[out] value An allocatable, unallocated 2d-double precision array
```

Comment:
This is the bit that needs updating.

Reply:
Fixed.

Reply:
You need to fix the 'allocatable' bit too :-)

Reply:
Fixed ... again